source/features/useful-not-found-page.tsx

```diff
@@ -145,13 +145,16 @@ async function showAlternateLink(): Promise<void> {
 
 function init(): false | void {
 	const parts = parseCurrentURL();
-	if (parts.length <= 1 || !select.exists('[alt*="This is not the web page you are looking for"]')) {
+	if (parts.length < 2) {
 		return false;
 	}
 
 	void showMissingPart();
-	void showDefaultBranchLink();
-	void showAlternateLink();
+
+	if (['tree', 'blob', 'edit'].includes(parts[2])) {
```

Comment:
If we had [a `required` field](https://github.com/sindresorhus/refined-github/issues/4008#issuecomment-812843149), I'd have rather used the regular `isSingleFile`/etc as suggested my comment

```js
	{
		include: [is404],
		require: [() => parseCurrentURL().length > 1],
		init: showMissingPart
	},
	{
		include: [is404],
		require: [
			isSingleFile,
			isTree,
			isEditingFile,
		],
		init: () => {showDefaultBranchLink(); showAlternateLink() }
	},
```